docs/source/models.rst

```diff
@@ -2654,6 +2654,58 @@ avoid failed inference requests.
 
 \***** If using `pyfunc` in MLflow Model Serving for realtime inference, the raw audio in bytes format must be base64 encoded prior to submitting to the endpoint. String inputs will be interpreted as uri locations.
 
+Using inference_config and dictionary parameters for `transformers` inference
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+For `transformers` inference, there are two ways to pass in additional arguments to the pipeline.
+
+* Use inference_config when saving/logging the model
+* Use dictionary parameters during `predict` function call
+
+.. note::
+    Dictionary parameters passed in to `predict` function will override the values in inference_config.
+    This functionality is not supported for model serving yet, but it will be supported in the short future.
+
+* Using inference_config
+
+.. code-block:: python
+
+    # Define an inference_config
+    inference_config = {
+        "num_beams": 5,
+        "max_length": 30,
+        "do_sample": True,
+        "remove_invalid_values": True,
+    }
+
+    # Saving inference_config with the model
+    mlflow.transformers.save_model(
+        model,
+        path=model_path,
+        inference_config=inference_config,
+        signature=signature,
+    )
+
+    pyfunc_loaded = mlflow.pyfunc.load_model(model_path)
+    # inference_config will be applied
+    result = pyfunc_loaded.predict(data)
+
+
+* Using dictionary parameters
+
+.. code-block:: python
+
+    # Saving model without inference_config
+    mlflow.transformers.save_model(
+        model,
+        path=model_path,
+        signature=signature,
+    )
+
+    pyfunc_loaded = mlflow.pyfunc.load_model(model_path)
+    # Passing inferenc_config dictionary as parameters to predict function directly
+    result = pyfunc_loaded.predict(data, parameters=inference_config)
```

Comment:
Is it possible to shorten `parameters` to `params?`

Reply:
That would change all signatures to params 🤔 Most cases users don't need to specify this `parameters=` as there're just two args for `predict` method

Reply:
`params` would be consistent with MLflow Tracking. Can we rename to `params` for consistency?

Reply:
@BenWilson2 agrees too

Reply:
Ok!